python/paddle/fluid/layers/detection.py

```diff
@@ -2221,3 +2222,59 @@ class number
     output.stop_gradient = True
 
     return output
+
+
+def collect_fpn_proposals(multi_rois,
+                          multi_scores,
+                          min_level,
+                          max_level,
+                          post_nms_topN,
+                          name=None):
+    """
+    Collect multi-level rois and select N rois with respect to multi_scores.
+    This operation performs as following steps:
+
+    1. Choose num_level rois and scores 
+       as input: num_level = max_level - min_level
+    2. Concat multi-level rois and scores
+    3. Sort scores and select post_nms_topN scores
+    4. Gather rois by selected index from scores
```

Comment:
```suggestion
    4. Gather rois by selected indexes from scores
```

should be plural form index ==> indexes / indices

Reply:
Done